modules/vaos/app/controllers/vaos/direct_scheduling_facilities_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
+
+module VAOS
+  class DirectSchedulingFacilitiesController < ApplicationController
+    def index
+      response = systems_service.get_system_facilities(
+        current_user,
+        facilities_params[:system_id],
+        facilities_params[:parent_code],
+        facilities_params[:type_of_care_id]
+      )
+
+      render json: VAOS::DirectSchedulingFacilitySerializer.new(response)
+    end
+
+    private
+
+    def systems_service
+      VAOS::SystemsService.new
+    end
+
+    def facilities_params
+      params.require(:parent_code)
+      params.require(:system_id)
+      params.require(:type_of_care_id)
+      params.permit(
+        :parent_code,
+        :system_id,
+        :type_of_care_id
+      )
```

Comment:
Looking at the cassette, a succesful request could be `?clinical-service=323&facility-code=688&parent-code=688`

And looking at this all are required, so could this be rewritten as 

`params.require(:parent_code, :system_id, :type_of_care_id).permit!` ?